src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -133,6 +135,17 @@ public Path getAndroidNDKPath() throws IOException {
         return resolvePath(configuration.getAndroidNdkPath(),"Fatal error, could not install Android NDK ");
     }
 
+    /**
+     * Return the path to the sysroot for this configuration.
+     * The path is cached on the environment variable.
+     * If it is not there yet, all dependencies are retrieved.
+     * @return the location of the sysroot for the arch-os for this configuration
+     * @throws IOException in case anything goes wrong.
+     */
+    public Path getSysrootPath() throws IOException {
+        return resolvePath(configuration.getSysrootPath(),"Fatal error, could not install sysroot zip");
```

Comment:
this doesn't add the architecture (e.g. it returns .gluon/substrate/sysroot instead of .gluon/substrate/sysroot/aarch64)

Reply:
Ok, fixed